pkg/OutputColorizer.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
         <id>OutputColorizer</id>
-        <version>1.0</version>
+        <version>1.0-rc2-1</version>
```

Comment:
This version was more of a test. We can take a closer look since perhaps we can keep the 1.0 version if NuGet has some sort of parameter to ignore that the dependencies are not RTM yet.